Dockerfile

```diff
@@ -1,33 +1,14 @@
-FROM golang:1.10.1-stretch as build
-RUN go get -u github.com/kisielk/errcheck &&\
-    go get -u honnef.co/go/tools/cmd/megacheck &&\
-    go get -u github.com/golang/lint/golint
+FROM golang:1.10-alpine AS gobuild-base
```

Comment:
>golang:1.10-alpine [](start = 5, length = 18)

doesn't golang add add git automatically?

Reply:
the alpine doesn't